package.json

```
@@ -14,19 +18,27 @@
     ],
     "*.{scss,html,js}": [
       "yarn lint:copyright --fix"
+    ],
+    "*.{tsx,ts,jsx,js}": [
+      "prettier --write",
+      "node packages/configs/copyrightLinter.js --fix"
     ]
   },
   "scripts": {
     "build": "turbo run build",
     "dev": "turbo run dev",
-    "test": "turbo run test --",
+    "test": "turbo run test",
     "lint": "turbo run lint",
+    "format": "turbo run format",
     "prepare": "husky install",
     "lint:copyright": "node packages/shared/copyrightLinter.js",
     "push-docker": "yarn workspace @itwin/itwinui-css push-docker",
     "createComponent": "yarn workspace @itwin/itwinui-css createComponent",
     "approve": "yarn workspace @itwin/itwinui-css approve",
```

Adding `:css` would be good to avoid confusion. Also, can we add one for react to make it simpler?

```suggestion
    "approve:css": "yarn workspace @itwin/itwinui-css approve",
    "approve:react": "yarn workspace storybook test:approve",
```